src/decision_process/examples.md

```diff
@@ -0,0 +1,297 @@
+# Examples of the decision-making process in action
+
+## Reversible decision: merging a proposal
+
+The process is best described by example. Suppose that there is a pending lang
+team proposal, and a lang team member would like to serve as the liaison. They
+contact the team leads and receive the go-ahead. They can then write:
+
+> @rustbot merge
+>
+> I propose to merge this proposal. I think it will be a great addition to
+> Rust!
+
+This indicates that they would like to merge the proposal. At the moment, there
+is no decision pending, so rustbot would add a comment that looks like the
+following:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member places a "hold" on the decision (or
+> cancels it).
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | |
+> | @Grace | |
+> | @Niklaus | |
+
+As the comment says, the PR is now in "pending decision" state, with Alan
+having kicked off the process with a proposal to merge. Alan's status of
+**merge** will link to his comment.
+
+Now, for this particular proposal, Barbara has a concern. She thinks that the
+proposal has overlooked an important consideration. She writes a comment:
+
+> @rustbot hold
+>
+> Did you consider reversing the polarity? Or the impact on the flux capacitor?
+
+At this point, rustbot updates the state:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | **hold** |
+> | @Grace | |
+> | @Niklaus | |
+
+Alan is currently busy at work, though, so by by the time that he and Barbara
+get a chance to talk, 11 days have passed. (Alan and Barbara receive a ping
+from rustbot after a week or so.) Once they get a chance to talk, Alan
+fully addresses Barbara's concern, so Barbara posts:
+
+> @rustbot merge
+
+The comment is updated:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | ~~hold~~ **merge** |
+> | @Grace | |
+> | @Niklaus | |
+
+Barbara's previous ~~hold~~ status links to her previous comment setting her
+status to `hold`, and her current **merge** status links to her more recent
+comment setting her status to `merge`.
+
+At this point, all the statuses are either empty, **abstain**, or **merge**,
```

Comment:
(no instances of "abstain" appear in the example as written. Its not a problem, per se, since I can infer how it would work. But the text might be clearer if you added an instance of it for Grace or Niklaus...)